src/app/(nousSuivre)/actions-solidaires/page.js

```
@@ -1,7 +1,218 @@
+
+            <motion.div
+                initial="hidden"
+                animate="visible"
+                variants={fadeIn}
+                className="bg-yellow-200 p-8"
+            >
+                <div className="items-center gap-8 xl:px-80">
+                    <div className="text-red-700 text-2xl md:text-4xl font-bold my-8">
+                        Actions <br /> Diverses
+                    </div>
+                    <p> cLe pôle Actions Solidaires propose divers services sociaux, incluant la circoncision, le soutien aux familles pendant l'Aid al-Adha, et inculquer à nos entants du foyer Bab Rayan nos valeurs solidaires par le biais de partage de dons en nature, reçus de nos partenaires et donateurs, aux familles en besoins.
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)